rendering/markup_content_test.go

```diff
@@ -0,0 +1,33 @@
+package rendering
+
+import "testing"
+import "github.com/stretchr/testify/assert"
+import "github.com/stretchr/testify/require"
+
+func TestGetDefaultMarkupFromNil(t *testing.T) {
```

Comment:
@kwk why would I do that ? I want short unit tests that follow as much as possible the "given/when/then" pattern, which is why I have 3 tests here.